batchflow/research/research.py

```diff
@@ -178,7 +184,7 @@ def load_results(self, *args, **kwargs):
         """ Load results of research as pandas.DataFrame or dict (see Results.load). """
         return Results(research=self).load(*args, **kwargs)
 
-    def _create_jobs(self, n_reps, n_iters, branches, name):
+    def _create_jobs(self, n_reps, n_iters, cv_splits, branches, name):
```

Comment:
```suggestion
    def _create_jobs(self, n_reps, n_iters, cv, branches, name):
```
Isn't it better or less clear??